gnucash/gnome/gnc-plugin-page-account-tree.c

```diff
@@ -1669,6 +1669,11 @@ gnc_plugin_page_account_tree_cmd_delete_account (GtkAction *action, GncPluginPag
           gnc_account_n_children (account)))
     {
         do_delete_account (account, NULL, NULL, NULL);
+
+        /* update opening balance account */
+        gnc_find_or_create_equity_account (gnc_account_get_root(account),
+                                           EQUITY_OPENING_BALANCE,
+                                           xaccAccountGetCommodity (account));
```

Comment:
@rhabacker Why would you create an opening balance account when deleting the account that might or might not use it?

Reply:
If the account to be deleted is the only opening balance account, I think a default account should be created.